docs/scenarios_authoring/DataSourcesAndSinks.md

```diff
@@ -9,145 +9,29 @@ sidebar_position: 3
-
-The disadvantage of using Kafka-typed-json as a data source is that whenever the `audit` topic will be used in the Kafka-typed-json source, its schema will have to be defined. This works well if there is only one occurrence of an audit topic in all Nussknacker scenarios; it may become difficult to maintain otherwise. 
-
-
-### No type information + JSON serialization
-
-There is only one component in this group -  Kafka-json source. The content of the JSON formatted Kafka event does not need to be declared in Nussknacker Designer. The values of the JSON object can still be accessed using the `#input.field-name `notation, however no hints will be available and no validations related to field names and their data types will be performed. 
-
-
-### Summary and comparison
-
-| Nussknacker source / sink component      | Schema description                          | Data serialization format  | Data types supported         | Limitations                                                                      | Advantages                                                        |
-|--------------------------------------    | --------------------                        | -------------------------- | ---------------------        | ----------------                                                                 | ---------------------------                                       |
-| Schema Registry + Avro serialization     | Avro in Confluent Schema Registry           | Avro                       | Avro supported               | Slightly more involving initially because of Schema Registry                     | <p> Hints and data type related validations </p> Maintainability  |
-| Schema Registry + JSON serialization     | Avro in Confluent Schema Registry           | JSON                       | Avro supported               | Slightly more involving initially because of Schema Registry                     | <p> Hints and data type related validations </p> Maintainability  |
-| Manually typed + JSON serialization      | Declared inside  Kafka-typed-json component | JSON                       | Common between Java and JSON | <p> May become difficult to maintain </p> No validation of input data at runtime | Hints during authoring                                            |
-| No type information + JSON serialization | none                                        | JSON                       | Common between Java and JSON | No hints and data type related validations                                       | Good for JSON objects whose structure is not known at design time |
-
- 
-
+If you prefer to define the whole output event as one expression containing value to be written to the Kafka topic, instead of filling separete autogenerated fields, you can do that by switching `Raw editor` to `true`.
```

Comment:
When using `Raw editor = true` one should also choose `Validation mode`. Maybe we should describe it too.

strict - Requires providing all fields, including optional fields, without redundant fields
lax - Requires providing only required fields, without optional fields, with redundant fields

Reply:
Added